hw1/02-palindrome.js

```diff
@@ -1,3 +1,53 @@
-const elem = document.querySelector('input');
+/**
+ * Checks if a string is a positive number or not.
+ *
+ * Examples:
+ *
+ * '0'    : false \
+ * 'abc'  : false \
+ * '-1'   : false \
+ * '001'  : true \
+ * '1.23' : true \
+ * 'Infinity' : true
+ */
+const isPos = function isPositiveNumber(str) {
```

Comment:
Hey Miles,

My bad on the delayed response I wasn't getting a reply option on your comment but turns out Its because I wasn't logged in. To comment on lines of code and do a code review when you navigate to a pull request there is a tab called "Files changed" just above the PR owners comment click this tab. This takes you to a page that shows you all the changes of the PR but if you hover your mouse just to the right of the code line number you will see a blue/white plus sign symbol pop up, if you click that you will then be able to leave a comment on that specific line of code. Do this for all the comments you want then make sure to click the green finish review button near the top right corner of the page. Happy to demonstrate as showing it might be better than than a wordy paragraph just let me know.